include/rest-request.php

```diff
@@ -122,4 +124,20 @@ public function set_language( $result, $server, $request ) {
 
 		return $result;
 	}
+
+	/**
+	 * Returns the locale based on current language.
+	 *
+	 * @since 3.7
+	 *
+	 * @param string $locale The current ID.
+	 * @return string
+	 */
+	public function get_locale( $locale ) {
+		if ( ! empty( $this->curlang ) ) {
+			return $this->curlang->locale;
+		}
+
+		return $locale;
```

Comment:
```suggestion
		if ( empty( $this->curlang ) ) {
			return $locale;
		}

		return $this->curlang->locale;
```

Please put the unhappy path in the conditional.

[_source_](https://github.com/szepeviktor/very-clean-code#-happy-and-unhappy-path)